app/views/popups/_add.html.slim

```diff
@@ -0,0 +1,22 @@
+.modal-header
+	button type="button" class="close" data-dismiss="modal" aria-label="Close"
+		span aria-hidden="true" &times;
+	h4.modal-title  Add a new Post or Project
+
+.modal-body.form-horizontal
+	.form-group
+		label.col-md-3 Add a
+		.col-md-9 
+			select.selectpicker.form-control.add_picker
+				option(value="posts") Post
+				option(value="projects") Project
+
+	img.loading_icon(src=image_path("loading.gif") style="display:none;")
```

Comment:
inline styles == bad. just make a global `hidden` class that you can use whenever you want `display:none`, then it's also super easy to change with javascript by just doing `$('.whatever-element').toggle('hidden')`